src/zcash/address/zip32.cpp

```diff
@@ -153,13 +297,73 @@ SaplingExtendedFullViewingKey SaplingExtendedSpendingKey::ToXFVK() const
     return ret;
 }
 
-libzcash::SaplingPaymentAddress SaplingExtendedSpendingKey::DefaultAddress() const
-{
-    return ToXFVK().DefaultAddress();
+//
+// Unified
+//
+
+std::optional<std::pair<UnifiedSpendingKey, HDKeyPath>> UnifiedSpendingKey::ForAccount(const HDSeed& seed, uint32_t bip44CoinType, uint32_t accountId) {
+    UnifiedSpendingKey usk;
+    usk.accountId = accountId;
+
+    auto transparentKey = DeriveZip32TransparentAccountKey(seed, bip44CoinType, accountId);
```

Comment:
Yep, confirmed that this function is being used for USKs, and thus is not implementing ZIP 316. Function should be renamed and fixed:
```suggestion
    auto transparentKey = DeriveBip44TransparentAccountKey(mnemonic, bip44CoinType, accountId);
```